pkg/cmd/repo/edit/edit.go

```diff
@@ -66,22 +66,37 @@ type EditOptions struct {
-	AllowUpdateBranch   *bool   `json:"allow_update_branch,omitempty"`
-	DefaultBranch       *string `json:"default_branch,omitempty"`
-	DeleteBranchOnMerge *bool   `json:"delete_branch_on_merge,omitempty"`
-	Description         *string `json:"description,omitempty"`
-	EnableAutoMerge     *bool   `json:"allow_auto_merge,omitempty"`
-	EnableIssues        *bool   `json:"has_issues,omitempty"`
-	EnableMergeCommit   *bool   `json:"allow_merge_commit,omitempty"`
-	EnableProjects      *bool   `json:"has_projects,omitempty"`
-	EnableDiscussions   *bool   `json:"has_discussions,omitempty"`
-	EnableRebaseMerge   *bool   `json:"allow_rebase_merge,omitempty"`
-	EnableSquashMerge   *bool   `json:"allow_squash_merge,omitempty"`
-	EnableWiki          *bool   `json:"has_wiki,omitempty"`
-	Homepage            *string `json:"homepage,omitempty"`
-	IsTemplate          *bool   `json:"is_template,omitempty"`
-	Visibility          *string `json:"visibility,omitempty"`
+	enableAdvancedSecurity             *bool
+	enableSecretScanning               *bool
+	enableSecretScanningPushProtection *bool
+
+	AllowForking        *bool                     `json:"allow_forking,omitempty"`
+	AllowUpdateBranch   *bool                     `json:"allow_update_branch,omitempty"`
+	DefaultBranch       *string                   `json:"default_branch,omitempty"`
+	DeleteBranchOnMerge *bool                     `json:"delete_branch_on_merge,omitempty"`
+	Description         *string                   `json:"description,omitempty"`
+	EnableAutoMerge     *bool                     `json:"allow_auto_merge,omitempty"`
+	EnableIssues        *bool                     `json:"has_issues,omitempty"`
+	EnableMergeCommit   *bool                     `json:"allow_merge_commit,omitempty"`
+	EnableProjects      *bool                     `json:"has_projects,omitempty"`
+	EnableDiscussions   *bool                     `json:"has_discussions,omitempty"`
+	EnableRebaseMerge   *bool                     `json:"allow_rebase_merge,omitempty"`
+	EnableSquashMerge   *bool                     `json:"allow_squash_merge,omitempty"`
+	EnableWiki          *bool                     `json:"has_wiki,omitempty"`
+	Homepage            *string                   `json:"homepage,omitempty"`
+	IsTemplate          *bool                     `json:"is_template,omitempty"`
+	SecurityAndAnalysis *SecurityAndAnalysisInput `json:"security_and_analysis,omitempty"`
+	Visibility          *string                   `json:"visibility,omitempty"`
+}
+
+type SecurityAndAnalysisInput struct {
+	EnableAdvancedSecurity             *SecurityAndAnalysisStatus `json:"advanced_security,omitempty"`
+	EnableSecretScanning               *SecurityAndAnalysisStatus `json:"secret_scanning,omitempty"`
+	EnableSecretScanningPushProtection *SecurityAndAnalysisStatus `json:"secret_scanning_push_protection,omitempty"`
+}
+
+type SecurityAndAnalysisStatus struct {
+	Status *string `json:"status,omitempty"`
```

Comment:
Consider grouping this bit with the helper function mentioned below. Logically, that might help with understanding the code in this section a bit better